.vimrc

```diff
@@ -84,8 +84,10 @@ set cursorline
 
 ""------------ Mappings --------------"
 "" Create tree navigation
-map <C-n> :vsp . <bar> vertical resize 30 <cr>
+map <C-n> :Lexplore <CR>
```

Comment:
What's your netrw version? Lexplore is available since netrw v150.

On Mon, Oct 19, 2015 at 11:21 AM, Carlos Oliveira <carlospecter@gmail.com>
wrote:

> My problem with Vexplore is that it's not toggleable. That's why I've used
> Lexplore
>
> On Fri, Oct 16, 2015 at 1:41 PM, Rodrigo Siqueira de Melo <
> notifications@github.com> wrote:
>
>> In .vimrc
>> <https://github.com/rodrigosiqueira/lappis_vimrc/pull/11#discussion_r42262806>
>> :
>>
>> > @@ -84,8 +84,10 @@ set cursorline
>> >
>> >  ""------------ Mappings --------------"
>> >  "" Create tree navigation
>> > -map <C-n> :vsp . <bar> vertical resize 30 <cr>
>> > +map <C-n> :Lexplore <CR>
>>
>> I'm using vim. I did a research and I found Vexplore, and it's works well
>> for me. Can you try on your computer?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/rodrigosiqueira/lappis_vimrc/pull/11/files#r42262806>
>> .
>>
>
>
>
> --
> Carlos Oliveira
> LAPPIS - UnB/FGA(https://lappis.unb.br/)
> GitHub: @carloshfoliveira <https://github.com/carloshfoliveira>
> LinkedIn: @carloshenriquef <https://br.linkedin.com/in/carloshenriquef>
>



-- 
Carlos Oliveira
LAPPIS - UnB/FGA(https://lappis.unb.br/)
GitHub: @carloshfoliveira <https://github.com/carloshfoliveira>
LinkedIn: @carloshenriquef <https://br.linkedin.com/in/carloshenriquef>